src/Jms/YamlValidatorConverter.php

```diff
@@ -0,0 +1,258 @@
+                                'LessThanOrEqual' => $item['value']
+                            ];
+                        }
+                        break;
+                    case 'minExclusive':
+                        foreach ($check as $item) {
+                            $rules[] = [
+                                'GreaterThan' => $item['value']
+                            ];
+                        }
+                        break;
+                    case 'minInclusive':
+                        foreach ($check as $item) {
+                            $rules[] = [
+                                'GreaterThanOrEqual' => $item['value']
+                            ];
+                        }
+                        break;
+                }
+            }
+
+            if (!!count($rules)) {
+
+                if ($arrayized){
+                    $rules = [
+                        ['All' => $rules]
+                    ];
+                }
+                $property["validator"] = $rules;
+            }
+        }
+
+    }
+
+    private function loadValidatorElement(array &$property, ElementItem $element, $arrayize)
+    {
+        /* @var $element Element */
+        $type = $element->getType();
+
+
+        $arrayized = false;
+        if ($arrayize) {
+
+            $attrs = [];
+            if ($itemOfArray = $this->isArrayNestedElement($type)) {
+                $attrs = [
+                    'min' => $itemOfArray->getMin(),
+                    'max' => $itemOfArray->getMax()
+                ];
+                $arrayized = true;
+            } elseif ($itemOfArray = $this->isArrayType($type)) {
+                $attrs = [
+                    'min' => $itemOfArray->getMin(),
+                    'max' => $itemOfArray->getMax()
+                ];
+                $arrayized = true;
+            } elseif ($this->isArrayElement($element)) {
+                $attrs = [
+                    'min' => $element->getMin(),
+                    'max' => $element->getMax()
+                ];
+                $arrayized = true;
+            }
+
+            if (count($attrs)) {
+                if ($attrs['min'] === 0) {
+                    unset($attrs['min']);
+                }
+                if ($attrs['max'] === -1) {
+                    unset($attrs['max']);
+                }
+                if (count($attrs) !== 0) {
+                    $property["validator"][] = [
+                        'Count' => $attrs
+                    ];
+                }
+            }
+
+        }
+
+        $this->loadValidatorType($property, $type, $arrayized);
+
+        // Required properties
+        if ($classType = $this->visitType($type)) {
+            if ($element->getMin() !== 0) {
+                if ($arrayized){
+                    $property["validator"][] = [
+                        'Count' => ['min' => 1]
+                    ];
+                }
+                $property["validator"][] = [
```

Comment:
At first is necessary for simple type/native type for indicate if required or not.